revisar-server-utils/config.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+import env from 'dotenv';
+

```

Comment:
does it work?
I think that it requires that every micro service provide this env variable.
Maybe you can make the auth middleware function receive a JWT token.
I want to consult @chenyair

Reply:
"Maybe you can make the auth middleware function receive a JWT token." - I did it at first.
But then i understood that in this implementation the config load the envs from the specific microservice that uses the shared util.
So if it used from reviews receiver it load his ACCESS_TOKEN_SECRET, and when it used from the backend it load the ACCESS_TOKEN_SECRET of the backend